subgraphs/spotify/src/index.ts

```diff
@@ -0,0 +1,114 @@
+import { readFileSync } from "fs";
+import gql from "graphql-tag";
+import { buildSubgraphSchema } from "@apollo/subgraph";
+import { ApolloServer, GraphQLResponse } from "@apollo/server";
+import { expressMiddleware } from "@apollo/server/express4";
+import resolvers from "./resolvers";
+import { ContextValue } from "./types/ContextValue";
+const port = process.env.PORT ?? "4001";
+const subgraphName = require("../package.json").name;
```

Comment:
Out of curiosity, any reason we can't use `import` here?

Reply:
Copy/pasta accident. We're not using that anyways so I went ahead and removed it